types/taskCard.type.ts

```diff
@@ -9,6 +10,7 @@ interface TaskCardProps {
     taskId: number;
     body: { name: string; description: string; done: boolean };
   }) => void;
+  onClick?: (param: any) => void;
 }
```

Comment:
any 타입 지양을 위해서 해당 타입으로 해주셔도 좋을것같습니다
```suggestion
  onClick?: React.Dispatch<React.SetStateAction<ITask | null>>;
```